### PR TITLE
fixes tool colors

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -489,17 +478,18 @@ obj/structure/cable/proc/cableColor(var/colorC)
 	stacktype = /obj/item/stack/cable_coil
 	drop_sound = 'sound/items/drop/accessory.ogg'
 	pickup_sound = 'sound/items/pickup/accessory.ogg'
+	var/possible_cable_coil_colours = list(
```
Comment (NonQueueingMatt) [inline code/modules/power/cable.dm:481]:
```suggestion
	var/list/possible_cable_coil_colours = list(
```

File: code/modules/power/cable.dm [hunk 2]
```diff
@@ -489,17 +478,18 @@ obj/structure/cable/proc/cableColor(var/colorC)
...
+		"Cyan" = COLOR_CYAN,
+		"Red" = COLOR_RED
+	)
 
-/obj/item/stack/cable_coil/Initialize(mapload, amt, param_color = null)
+/obj/item/stack/cable_coil/Initialize(mapload, amt)
 	. = ..(mapload, amt)
-	icon_state = "coil"
-	our_color = pick(possible_cable_coil_colours)
-	add_atom_colour(possible_cable_coil_colours[our_color])
```
Comment (NonQueueingMatt) [inline code/modules/power/cable.dm:497]: You don't use this proc anymore, do you? If so, remove it

**Pull request merged**